### ISSUE TITLE
Appears to be a git repo or submodule.

### ISSUE THREAD

**Issue by ahunter135** [rogierschouten/ts-events#15]
When I try to do `npm i` on a project that has ts-events, I get a npm error.

`node_modules/ts-events: Appears to be a git repo or submodule. Refusing to remove it. Update manually, or move it out of the way first.`

I am assuming this occurred with the latest commit?

**Comment by rogierschouten** [comment]
I cannot reproduce that, and installing ts-events does not produce a .git directory.  Could it be that you accidentally got a .git directory into the ts-events folder? 

If not, can I have your package.json or a link to your repo?

**Comment by Hexxeh** [comment]
```
$ npm pack ts-events
npm notice 
npm notice 📦  ts-events@3.3.0
npm notice === Tarball Contents === 
npm notice 2.6kB   package.json              
npm notice 760B    LICENSE                   
npm notice 18.8kB  README.md                 
npm notice 177B    .git/hooks/README.sample  
npm notice 4.9kB   dist/lib/any-event.d.ts   
npm notice 38.1kB  dist/lib/any-event.js     
npm notice 2.6kB   dist/lib/async-event.d.ts 
npm notice 19.3kB  dist/lib/async-event.js   
npm notice 4.0kB   dist/lib/base-event.d.ts  
npm notice 22.0kB  dist/lib/base-event.js    
npm notice 1.7kB   dist/lib/EventQueue.d.ts  
npm notice 12.0kB  dist/lib/EventQueue.js    
npm notice 1.1kB   dist/lib/index.d.ts       
npm notice 4.3kB   dist/lib/index.js         
npm notice 64B     dist/lib/objects.d.ts     
npm notice 7.3kB   dist/lib/objects.js       
npm notice 2.1kB   dist/lib/queued-event.d.ts
npm notice 14.7kB  dist/lib/queued-event.js  
npm notice 1.5kB   dist/lib/sync-event.d.ts  
npm notice 9.4kB   dist/lib/sync-event.js    
npm notice 148.3kB dist/ts-events.js         
npm notice 9B      dist/ts-events.min.js     
npm notice === Tarball Details === 
npm notice name:          ts-events                               
npm notice version:       3.3.0                                   
npm notice filename:      ts-events-3.3.0.tgz                     
npm notice package size:  73.4 kB                                 
npm notice unpacked size: 315.7 kB                                
npm notice shasum:        7ce5af829627faad991c5804f318ae074fcac4e7
npm notice integrity:     sha512-1c2xrmh2HKBcO[...]6C7dYHePwZC+A==
npm notice total files:   22                                      
npm notice 
ts-events-3.3.0.tgz
```

There's a `.git` folder in the tarball published to npm.

**Comment by rogierschouten** [comment]
Thanks @Hexxeh and @ahunter135 

Fixed in v3.3.1.  Please let me know if there are any issues.
